modules/rosa-sts-creating-account-wide-sts-roles-and-policies.adoc

```diff
@@ -20,9 +29,19 @@ Before using {cluster-manager-first} to create {product-title} (ROSA) clusters t
 To successfully install ROSA clusters, use the latest version of the ROSA CLI.
 ====
 * You have logged in to your Red Hat account by using the `rosa` CLI.
+endif::[]
 
 .Procedure
 
+ifdef::quick-install[]
+. Check your AWS account for existing roles and policies:
++
+[source,terminal]
+----
+$ rosa list account-roles
+----
+endif::[]
+
 . If they do not exist in your AWS account, create the required account-wide STS roles and policies, including the Operator policies:
```

Comment:
remove:
', including the Operator policies'

because they are created in a later step.